src/components/SIngleArticle.jsx

```diff
@@ -26,23 +26,19 @@ function SingleArticle() {
       });
   }, [article_id]);
 
-  function increaseVotes(comment) {
-    const updatedComments = comments.map((singleComment) => {
-      if (singleComment.comment_id === comment.comment_id) {
-        return { ...singleComment, votes: singleComment.votes + 1 };
-      }
-      return singleComment;
-    });
-    setComments(updatedComments);
+  function increaseArticleVotes() {
+    const updatedArticle = { ...article, votes: article.votes + 1 };
+    setArticle(updatedArticle);
+    axios.patch(`https://ncn-network.onrender.com/api/articles/${article_id}`,{
```

Comment:
what happens to the displayed number of votes if the patch request errors and the votes are not updated on the db? What should the user see as we have already shown them the vote going up. Same for the downvote too